client/src/component/signin/index.jsx

```diff
@@ -0,0 +1,128 @@
+import { React, useState } from 'react';
+import {
+  Layout,
+  Row,
+  Col,
+  Image,
+  Form,
+  Button,
+  Input,
+  message,
+} from 'antd';
+
+import axios from 'axios';
+import 'antd/dist/antd.css';
+import './index.css';
+
+const {
+  Content,
+} = Layout;
+
+function Signin() {
+  const [userData, setUserData] = useState({ email: '', password: '' });
+
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setUserData({
+      ...userData,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async () => {
+    try {
+      // Post data here
+      await axios.post('/api/signin', userData);
+      message.success('Welcome back!');
+    } catch (e) {
+      message.warning('Try again');
+    }
```

Comment:
We already send an error message in the back so why not use it here? Imagine you are trying to log in to a website and it continues saying try again, so didn't know exactly the problem.